dll/win32/uxtheme/uxthemep.h

```diff
@@ -33,6 +33,10 @@ WINE_DEFAULT_DEBUG_CHANNEL(uxtheme);
 #define MAX_THEME_CLASS_NAME 60
 #define MAX_THEME_VALUE_NAME 60
 
+#if defined(__REACTOS__) || _WIN32_WINNT >= 0x600
```

Comment:
```suggestion
#if defined(__REACTOS__) || _WIN32_WINNT >= _WIN32_WINNT_VISTA
```

Reply:
or even `_WIN32_WINNT_LONGHORN` in case this was used there.

Reply:
@Splitwirez yes exactly. Post-reset Longhorn is what we usually designate with the `_WIN32_WINNT_LONGHORN`. (But granted in terms of these macros it's just a matter of "description" since both the LONGHORN and the VISTA ones have the same numerical value.)